src/main/java/com/kichik/pecoff4j/util/RCEdit.java

```diff
@@ -1,89 +0,0 @@
-/*******************************************************************************
- * This program and the accompanying materials
- * are made available under the terms of the Common Public License v1.0
- * which accompanies this distribution, and is available at 
- * http://www.eclipse.org/legal/cpl-v10.html
- * 
- * Contributors:
- *     Peter Smith
- *******************************************************************************/
-package com.kichik.pecoff4j.util;
-
-import java.io.File;
-import java.io.IOException;
-
-import com.kichik.pecoff4j.PE;
-import com.kichik.pecoff4j.ResourceDirectory;
-import com.kichik.pecoff4j.io.PEParser;
-
-public class RCEdit {
-	public static void main(String[] args) throws Exception {
-		launch(new String[0]);
-		launch(new String[] { "/I", "test/WinRun4J.exe", "test/eclipse.ico" });
```

Comment:
Do we have a test to replace this?

Reply:
Sadly, this class is only a stub. The `printUsage` method documents what the authors of WinRun4J planned to achieve and some methods actually load the executable, but do nothing more.
When I discovered this library, I was also disappointed when realizing that it does not hold its promises... ;-)